docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,170 @@ description: Next.js supports built-in image optimization, as well as third part
+>### What if I don't know the size of my images?
+>
+>If you are accessing images from a source without knowledge of the images' sizes, there are several things you can do:
+>
+>**Use `layout='fill'`**
+>
+>The `fill` layout mode allows your image to be sized by its parent element. Consider using CSS to give the image's parent element space on the page, then using the [`objectFit property`](/docs/api-reference/next/image.md#objectfit) with `fill`, `contain`, or `cover`, along with the [`objectPosition property`](/docs/api-reference/next/image.md#objectposition) to define how the image should occupy that space. 
+>
+>**Normalize your images**
+>
+>If you're serving images from a source that you control, consider modifying your image pipeline to normalize the images to a specific size.
+>
+>**Modify your API calls**
+>
+>If your application is retrieving image URLs using an API call (such as to a CMS), you may be able to modify the API call to return the image dimensions along with the URL.
 
-The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header.
+If none of the suggested methods works for sizing your images, the `next/image` component is designed to work well on a page alongside standard `<img>` elements. 
 
-If `s-maxage` is found in `Cache-Control`, it is used. If no `s-maxage` is found, then `max-age` is used. If no `max-age` is found, then [`minimumCacheTTL`](#minimum-cache-ttl) is used.
+## Styling
 
-You can configure [`minimumCacheTTL`](#minimum-cache-ttl) to increase the cache duration when the upstream image does not include `max-age`.
+Styling the Image component is not that different from styling a norrmal `<img>` element, but there are a few guidelines to keep in mind:
 
-You can also configure [`deviceSizes`](#device-sizes) and [`imageSizes`](#device-sizes) to reduce the total number of possible generated images.
+**Pick the correct layout mode**
 
-## Advanced
+The Image component has several different [layout modes](/docs/api-reference/next/image.md#layout) that define how it is sized on the page. Familiarize yourself with the various modes, and consider experimenting with other modes if your styling isn't working as you think it should.
 
-The following configuration is for advanced use cases and is usually not necessary. If you choose to configure the properties below, you will override any changes to the Next.js defaults in future updates.
+**Target the image with className, not based on DOM structure**
 
-### Device Sizes
+The Image component has a somewhat more complicated DOM structure than a standard `<img>`. Regardless of layout mode used, the inner `<img>` element will always be wrapped by exactly one `<div>`. It may have a sibling `<div>` for spacing, depending on layout. These additional `<div>`s are what allows us to provide the different layout modes, while ensuring no layout shift.
```

Comment:
```suggestion
Regardless of the layout mode used, the Image component will have a consistent DOM structure of one `<img>` tag wrapped by exactly one `<div>`. For some modes, it may also have a sibling `<div>` for spacing. These additional `<div>` elements are critical to allow the component to prevent layout shifts.
```